### PR TITLE
✨ PRTL-1494 Replace pie chart in summarycards with bar charts on Case, Project entity pages and Cart page

### PR DESCRIPTION
**Pull request by cheapsteak**
Add spark meters to Case, Project entity pages and Cart page

image

image
image

### CODE REVIEW

File: src/packages/@ncigdc/containers/CartPage.js [hunk 1]
```diff
@@ -127,15 +153,86 @@ const CartPage: TCartPage = ({ viewer, files, user, theme } = {}) => {
                   style: { textAlign: 'right' },
                 },
                 {
+                  key: 'case_count_meter',
+                  title: (
+                    <Link
+                      merge="replace"
+                      pathname="/repository"
+                      query={{
+                        filters: makeFilter(
```
Comment (Jephuff) [inline src/packages/@ncigdc/containers/CartPage.js]: I'm not sure this abstraction is helpful :D 
I think writing the actual filter would be clearer
```
{
    op: 'or',
    content: files.map(file => ({
        op: 'in',
        content: {
            field: 'files.file_id',
            value: [file.file_id],
        }
    }))
}
```

**Comment by cheapsteak** [comment]
ready for re-review

File: src/packages/@ncigdc/uikit/SparkMeter.js [hunk 2]
```diff
@@ -10,10 +10,12 @@ export default ({ value, width = 30, max = 30 }) =>
       display: 'inline-block',
       pointerEvents: 'none',
     }}
+    {...props}
```
Comment (Jephuff) [inline src/packages/@ncigdc/uikit/SparkMeter.js]: Should merge this styles with `props.style`
Also either destructure style or move the prop spread to the top.

File: src/packages/@ncigdc/utils/filters/index.js [hunk 3]
```diff
@@ -214,10 +214,10 @@ export const getFilterValue = ({ currentFilters, dotField }) =>
 type TMakeFilter = (
   fields: [{ field: string, value: string }],
 ) => Object | string;
-export const makeFilter: TMakeFilter = fields => {
+export const makeFilter: TMakeFilter = (fields, { op } = { op: 'and' }) => {
```
Comment (Jephuff) [inline src/packages/@ncigdc/utils/filters/index.js]: Seeing as the op param isn't used any more, this could be reverted.

**Comment by cheapsteak** [comment]
Could I have another? Thanks @Jephuff

**Pull request merged**